ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -148,6 +149,19 @@ data:
       # We had to resort to a fixed list here because there are a lot of openshift-RAND namespaces created.
       # (must-gather, debug, amq, test, etc) If you want your namespaces to get it's own indexed namespace
       # label, add it here, in alphabetical order, and to the near identical line below with =~ instead of !~.
+      #
+      # Special case for KubeVirt virt-launcher pods, where we also pack the domain label (VM name)
+      - match:
+          selector: '{pod=~"virt-launcher-*"}'
```

Comment:
not sure we can already filter by pod here
if not, would selector: '{namespace=~"kubevirt*"}' be acceptable?

Reply:
No. Since usually virt-launcher pods are not created in kubevirt namespace.

Reply:
This loki instance is only for the CI, in the CI I think all tests happen in namespaces like "kubevirt-test-ns-1", stuff like that